examples/temporal/main.js

```diff
@@ -0,0 +1,592 @@
+( function () {
```

Comment:
Example should start from template example like example/quad-texture no global variable except the instance of the example and the deps. If you need access for debug you can still access member of the example instance.